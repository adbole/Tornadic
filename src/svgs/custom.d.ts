declare module "*.svg";
declare module "radar/*.svg";
declare module "conditions/*.svg"
declare module "widget/*.svg"