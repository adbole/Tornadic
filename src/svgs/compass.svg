<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="compass"
   width="240"
   height="240"
   version="1.1"
   sodipodi:docname="compass.svg"
   inkscape:version="1.2 (dc2aedaf03, 2022-05-15)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs362" />
  <sodipodi:namedview
     id="namedview360"
     pagecolor="#505050"
     bordercolor="#ffffff"
     borderopacity="1"
     inkscape:showpageshadow="0"
     inkscape:pageopacity="0"
     inkscape:pagecheckerboard="1"
     inkscape:deskcolor="#505050"
     showgrid="false"
     inkscape:zoom="1.9728279"
     inkscape:cx="78.56742"
     inkscape:cy="120.63901"
     inkscape:window-width="1920"
     inkscape:window-height="1017"
     inkscape:window-x="-8"
     inkscape:window-y="-8"
     inkscape:window-maximized="1"
     inkscape:current-layer="compass" />
  <text
     x="113"
     y="29.316406"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text183"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">N</text>
  <text
     x="213.41797"
     y="128"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text185"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">E</text>
  <text
     x="113"
     y="224.56055"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text187"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">S</text>
  <text
     x="14.755859"
     y="128"
     font-size="20px"
     font-family="Helvetica"
     fill="#ffffff"
     id="text189"
     style="-inkscape-font-specification:'Arial, Bold';font-family:Arial;font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;font-size:20px;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#ffffff">W</text>
  <line
     x1="120"
     y1="0"
     x2="120"
     y2="10"
     stroke="#ffffff"
     stroke-width="2"
     id="line191" />
  <line
     x1="130.45869"
     y1="0.45663643"
     x2="129.58713"
     y2="10.418583"
     stroke="#808080"
     stroke-width="1"
     id="line195" />
  <line
     x1="140.83778"
     y1="1.8230705"
     x2="139.1013"
     y2="11.671146"
     stroke="#808080"
     stroke-width="1"
     id="line197" />
  <line
     x1="151.05829"
     y1="4.0889015"
     x2="148.47009"
     y2="13.748158"
     stroke="#808080"
     stroke-width="1"
     id="line199" />
  <line
     x1="161.04242"
     y1="7.2368851"
     x2="157.62222"
     y2="16.633812"
     stroke="#808080"
     stroke-width="1"
     id="line201" />
  <line
     x1="170.71419"
     y1="11.243065"
     x2="166.48801"
     y2="20.306145"
     stroke="#808080"
     stroke-width="1"
     id="line203" />
  <line
     x1="180"
     y1="16.07695"
     x2="175"
     y2="24.737206"
     stroke="#ffffff"
     stroke-width="2"
     id="line205" />
  <line
     x1="188.82918"
     y1="21.701756"
     x2="183.09341"
     y2="29.893276"
     stroke="#808080"
     stroke-width="1"
     id="line209" />
  <line
     x1="197.13451"
     y1="28.074665"
     x2="190.70663"
     y2="35.735115"
     stroke="#808080"
     stroke-width="1"
     id="line211" />
  <line
     x1="204.85281"
     y1="35.147186"
     x2="197.78175"
     y2="42.218254"
     stroke="#808080"
     stroke-width="1"
     id="line213" />
  <line
     x1="211.92534"
     y1="42.865486"
     x2="204.26489"
     y2="49.293365"
     stroke="#808080"
     stroke-width="1"
     id="line215" />
  <line
     x1="218.29825"
     y1="51.17083"
     x2="210.10672"
     y2="56.906593"
     stroke="#808080"
     stroke-width="1"
     id="line217" />
  <line
     x1="223.92305"
     y1="60"
     x2="215.26279"
     y2="65"
     stroke="#ffffff"
     stroke-width="2"
     id="line219" />
  <line
     x1="228.75693"
     y1="69.285812"
     x2="219.69386"
     y2="73.511993"
     stroke="#808080"
     stroke-width="1"
     id="line223" />
  <line
     x1="232.76311"
     y1="78.957581"
     x2="223.3662"
     y2="82.377777"
     stroke="#808080"
     stroke-width="1"
     id="line225" />
  <line
     x1="235.9111"
     y1="88.941711"
     x2="226.25185"
     y2="91.529907"
     stroke="#808080"
     stroke-width="1"
     id="line227" />
  <line
     x1="238.17693"
     y1="99.162216"
     x2="228.32886"
     y2="100.8987"
     stroke="#808080"
     stroke-width="1"
     id="line229" />
  <line
     x1="239.54337"
     y1="109.54131"
     x2="229.58142"
     y2="110.41287"
     stroke="#808080"
     stroke-width="1"
     id="line231" />
  <line
     x1="240"
     y1="120"
     x2="230"
     y2="120"
     stroke="#ffffff"
     stroke-width="2"
     id="line233" />
  <line
     x1="239.54337"
     y1="130.45869"
     x2="229.58142"
     y2="129.58713"
     stroke="#808080"
     stroke-width="1"
     id="line237" />
  <line
     x1="238.17693"
     y1="140.83778"
     x2="228.32886"
     y2="139.1013"
     stroke="#808080"
     stroke-width="1"
     id="line239" />
  <line
     x1="235.9111"
     y1="151.05829"
     x2="226.25185"
     y2="148.47009"
     stroke="#808080"
     stroke-width="1"
     id="line241" />
  <line
     x1="232.76311"
     y1="161.04242"
     x2="223.3662"
     y2="157.62222"
     stroke="#808080"
     stroke-width="1"
     id="line243" />
  <line
     x1="228.75693"
     y1="170.71419"
     x2="219.69386"
     y2="166.48801"
     stroke="#808080"
     stroke-width="1"
     id="line245" />
  <line
     x1="223.92305"
     y1="180"
     x2="215.26279"
     y2="175"
     stroke="#ffffff"
     stroke-width="2"
     id="line247" />
  <line
     x1="218.29825"
     y1="188.82918"
     x2="210.10672"
     y2="183.09341"
     stroke="#808080"
     stroke-width="1"
     id="line251" />
  <line
     x1="211.92534"
     y1="197.13451"
     x2="204.26489"
     y2="190.70663"
     stroke="#808080"
     stroke-width="1"
     id="line253" />
  <line
     x1="204.85281"
     y1="204.85281"
     x2="197.78175"
     y2="197.78175"
     stroke="#808080"
     stroke-width="1"
     id="line255" />
  <line
     x1="197.13451"
     y1="211.92534"
     x2="190.70663"
     y2="204.26489"
     stroke="#808080"
     stroke-width="1"
     id="line257" />
  <line
     x1="188.82918"
     y1="218.29825"
     x2="183.09341"
     y2="210.10672"
     stroke="#808080"
     stroke-width="1"
     id="line259" />
  <line
     x1="180"
     y1="223.92305"
     x2="175"
     y2="215.26279"
     stroke="#ffffff"
     stroke-width="2"
     id="line261" />
  <line
     x1="170.71419"
     y1="228.75693"
     x2="166.48801"
     y2="219.69386"
     stroke="#808080"
     stroke-width="1"
     id="line265" />
  <line
     x1="161.04242"
     y1="232.76312"
     x2="157.62222"
     y2="223.3662"
     stroke="#808080"
     stroke-width="1"
     id="line267" />
  <line
     x1="151.05829"
     y1="235.9111"
     x2="148.47009"
     y2="226.25183"
     stroke="#808080"
     stroke-width="1"
     id="line269" />
  <line
     x1="140.83778"
     y1="238.17694"
     x2="139.1013"
     y2="228.32886"
     stroke="#808080"
     stroke-width="1"
     id="line271" />
  <line
     x1="130.45869"
     y1="239.54337"
     x2="129.58713"
     y2="229.58142"
     stroke="#808080"
     stroke-width="1"
     id="line273" />
  <line
     x1="120"
     y1="240"
     x2="120"
     y2="230"
     stroke="#ffffff"
     stroke-width="2"
     id="line275" />
  <line
     x1="109.54131"
     y1="239.54337"
     x2="110.41286"
     y2="229.58142"
     stroke="#808080"
     stroke-width="1"
     id="line279" />
  <line
     x1="99.162216"
     y1="238.17694"
     x2="100.8987"
     y2="228.32886"
     stroke="#808080"
     stroke-width="1"
     id="line281" />
  <line
     x1="88.941711"
     y1="235.9111"
     x2="91.529907"
     y2="226.25183"
     stroke="#808080"
     stroke-width="1"
     id="line283" />
  <line
     x1="78.957581"
     y1="232.76312"
     x2="82.377785"
     y2="223.3662"
     stroke="#808080"
     stroke-width="1"
     id="line285" />
  <line
     x1="69.285812"
     y1="228.75693"
     x2="73.511993"
     y2="219.69386"
     stroke="#808080"
     stroke-width="1"
     id="line287" />
  <line
     x1="60"
     y1="223.92305"
     x2="65"
     y2="215.26279"
     stroke="#ffffff"
     stroke-width="2"
     id="line289" />
  <line
     x1="51.170826"
     y1="218.29825"
     x2="56.906593"
     y2="210.10672"
     stroke="#808080"
     stroke-width="1"
     id="line293" />
  <line
     x1="42.865486"
     y1="211.92534"
     x2="49.293362"
     y2="204.26489"
     stroke="#808080"
     stroke-width="1"
     id="line295" />
  <line
     x1="35.147186"
     y1="204.85281"
     x2="42.218254"
     y2="197.78175"
     stroke="#808080"
     stroke-width="1"
     id="line297" />
  <line
     x1="28.074667"
     y1="197.13451"
     x2="35.735111"
     y2="190.70663"
     stroke="#808080"
     stroke-width="1"
     id="line299" />
  <line
     x1="21.701756"
     y1="188.82918"
     x2="29.893274"
     y2="183.09341"
     stroke="#808080"
     stroke-width="1"
     id="line301" />
  <line
     x1="16.076952"
     y1="180"
     x2="24.737206"
     y2="175"
     stroke="#ffffff"
     stroke-width="2"
     id="line303" />
  <line
     x1="11.243066"
     y1="170.71419"
     x2="20.306143"
     y2="166.48801"
     stroke="#808080"
     stroke-width="1"
     id="line307" />
  <line
     x1="7.2368855"
     y1="161.04242"
     x2="16.633812"
     y2="157.62222"
     stroke="#808080"
     stroke-width="1"
     id="line309" />
  <line
     x1="4.088901"
     y1="151.05829"
     x2="13.748159"
     y2="148.47009"
     stroke="#808080"
     stroke-width="1"
     id="line311" />
  <line
     x1="1.8230697"
     y1="140.83778"
     x2="11.671147"
     y2="139.1013"
     stroke="#808080"
     stroke-width="1"
     id="line313" />
  <line
     x1="0.45663622"
     y1="130.45869"
     x2="10.418583"
     y2="129.58713"
     stroke="#808080"
     stroke-width="1"
     id="line315" />
  <line
     x1="0"
     y1="120"
     x2="10"
     y2="120"
     stroke="#ffffff"
     stroke-width="2"
     id="line317" />
  <line
     x1="0.45663622"
     y1="109.54131"
     x2="10.418583"
     y2="110.41287"
     stroke="#808080"
     stroke-width="1"
     id="line321" />
  <line
     x1="1.8230697"
     y1="99.162216"
     x2="11.671147"
     y2="100.8987"
     stroke="#808080"
     stroke-width="1"
     id="line323" />
  <line
     x1="4.088901"
     y1="88.941711"
     x2="13.748159"
     y2="91.529907"
     stroke="#808080"
     stroke-width="1"
     id="line325" />
  <line
     x1="7.2368855"
     y1="78.957581"
     x2="16.633812"
     y2="82.377777"
     stroke="#808080"
     stroke-width="1"
     id="line327" />
  <line
     x1="11.243066"
     y1="69.285812"
     x2="20.306143"
     y2="73.511993"
     stroke="#808080"
     stroke-width="1"
     id="line329" />
  <line
     x1="16.076952"
     y1="60"
     x2="24.737206"
     y2="65"
     stroke="#ffffff"
     stroke-width="2"
     id="line331" />
  <line
     x1="21.701756"
     y1="51.17083"
     x2="29.893274"
     y2="56.906593"
     stroke="#808080"
     stroke-width="1"
     id="line335" />
  <line
     x1="28.074667"
     y1="42.865486"
     x2="35.735111"
     y2="49.293365"
     stroke="#808080"
     stroke-width="1"
     id="line337" />
  <line
     x1="35.147186"
     y1="35.147186"
     x2="42.218254"
     y2="42.218254"
     stroke="#808080"
     stroke-width="1"
     id="line339" />
  <line
     x1="42.865486"
     y1="28.074665"
     x2="49.293362"
     y2="35.735115"
     stroke="#808080"
     stroke-width="1"
     id="line341" />
  <line
     x1="51.170826"
     y1="21.701756"
     x2="56.906593"
     y2="29.893276"
     stroke="#808080"
     stroke-width="1"
     id="line343" />
  <line
     x1="60"
     y1="16.07695"
     x2="65"
     y2="24.737206"
     stroke="#ffffff"
     stroke-width="2"
     id="line345" />
  <line
     x1="69.285812"
     y1="11.243065"
     x2="73.511993"
     y2="20.306145"
     stroke="#808080"
     stroke-width="1"
     id="line349" />
  <line
     x1="78.957581"
     y1="7.2368851"
     x2="82.377785"
     y2="16.633812"
     stroke="#808080"
     stroke-width="1"
     id="line351" />
  <line
     x1="88.941711"
     y1="4.0889015"
     x2="91.529907"
     y2="13.748158"
     stroke="#808080"
     stroke-width="1"
     id="line353" />
  <line
     x1="99.162216"
     y1="1.8230705"
     x2="100.8987"
     y2="11.671146"
     stroke="#808080"
     stroke-width="1"
     id="line355" />
  <line
     x1="109.54131"
     y1="0.45663643"
     x2="110.41286"
     y2="10.418583"
     stroke="#808080"
     stroke-width="1"
     id="line357" />
</svg>
