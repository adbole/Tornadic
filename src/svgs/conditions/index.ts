export { ReactComponent as Drizzle } from "./cloud-drizzle-fill.svg";
export { ReactComponent as Fog } from "./cloud-fog-fill.svg";
export { ReactComponent as Lightning } from "./cloud-lightning-rain-fill.svg";
export { ReactComponent as CloudMoon } from "./cloud-moon-fill.svg";
export { ReactComponent as Rain } from "./cloud-rain-fill.svg";
export { ReactComponent as Cloud } from "./cloud-single-fill.svg";
export { ReactComponent as CloudSun } from "./cloud-sun-fill.svg";
export { ReactComponent as Clouds } from "./clouds-fill.svg";
export { ReactComponent as Moon } from "./moon-fill.svg";
export { ReactComponent as Snow } from "./snow.svg";
export { ReactComponent as Sun } from "./sun-fill.svg";