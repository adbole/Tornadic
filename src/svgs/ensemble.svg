<svg fill="currentColor" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
 <path d="m15.314 5.1134c0.21387 0.175 0.24522 0.49031 0.07 0.704l-4.5 5.5c-0.18727 0.22858-0.53086 0.24576-0.74 0.037l-2.587-2.587-3.656 5.027c-0.39701 0.50483-1.1663-0.05497-0.808-0.588l4-5.5c0.18054-0.2487 0.54059-0.2772 0.758-0.06l2.609 2.61 4.15-5.073c0.175-0.21387 0.49031-0.24522 0.704-0.07m-1-4c0.21387 0.175 0.24522 0.49031 0.07 0.704l-4.5 5.5c-0.18727 0.22858-0.53086 0.24576-0.74 0.037l-2.587-2.587-3.656 5.027c-0.39701 0.50483-1.1663-0.05497-0.808-0.588l4-5.5c0.18054-0.2487 0.54059-0.2772 0.758-0.06l2.609 2.61 4.15-5.073c0.175-0.21387 0.49031-0.24522 0.704-0.07m-14.314-1.1134h1v15h15v1h-16zm14.817 3.113c0.21387 0.175 0.24522 0.49031 0.07 0.704l-4.5 5.5c-0.18727 0.22858-0.53086 0.24576-0.74 0.037l-2.587-2.587-3.656 5.027c-0.39701 0.50483-1.1663-0.05497-0.808-0.588l4-5.5c0.18054-0.2487 0.54059-0.2772 0.758-0.06l2.609 2.61 4.15-5.073c0.175-0.21387 0.49031-0.24522 0.704-0.07"/>
</svg>
