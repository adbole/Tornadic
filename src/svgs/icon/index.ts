import {ReactComponent as Tornadic} from './tornadic.svg';
import {ReactComponent as TornadicFull} from './tornadic.svg';
import {ReactComponent as TornadicLoader} from './tornadic-loader.svg';

export {
    Tornadic,
    TornadicFull,
    TornadicLoader
};