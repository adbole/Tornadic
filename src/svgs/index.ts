/// <reference types="vite-plugin-svgr/client" />

export { default as Ensemble } from "./ensemble.svg?react";
export { default as ExclamationTriangle } from "./exclamation-triangle-fill.svg?react";
export { default as Spinner } from "./spinner.svg?react";
export { default as WifiOff } from "./wifi-off.svg?react";
