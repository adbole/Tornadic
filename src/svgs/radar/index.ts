import { ReactComponent as CircleSlashes } from "./circle-slashes.svg";
import { ReactComponent as LocationDot } from "./current-location-dot.svg";
import { ReactComponent as Cursor } from "./cursor-fill.svg";
import { ReactComponent as Grid } from "./grid.svg";
import { ReactComponent as Pause } from "./pause-fill.svg";
import { ReactComponent as Play } from "./play-fill.svg";

export {
    CircleSlashes,
    Cursor,
    Grid,
    LocationDot,
    Pause,
    Play 
};