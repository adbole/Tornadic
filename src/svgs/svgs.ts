import {ReactComponent as Tornadic} from './tornadic.svg'
import { ReactComponent as Play } from './play-fill.svg'
import { ReactComponent as Pause } from './pause-fill.svg'
import { ReactComponent as Grid } from './grid.svg'
import { ReactComponent as Cursor } from './cursor-fill.svg'


export { 
    Tornadic,
    Play,
    Pause,
    Grid,
    Cursor
};