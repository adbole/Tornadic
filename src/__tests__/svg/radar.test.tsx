import { render } from '@testing-library/react';

import * as svgs from 'svgs/radar';


test.each(Object.entries(svgs))('%s', (_, Svg) => {
  const comp = render(<Svg />);

  expect(comp).toMatchSnapshot();
});