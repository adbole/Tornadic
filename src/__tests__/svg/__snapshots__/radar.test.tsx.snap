// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CircleSlashes 1`] = `
<div>
  <svg
    class="bi bi-slash-circle"
    fill="currentColor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"
    />
    <path
      d="M11.354 4.646a.5.5 0 0 0-.708 0l-6 6a.5.5 0 0 0 .708.708l6-6a.5.5 0 0 0 0-.708z"
    />
    <path
      d="m8.3537 3.6459c-0.19536-0.19586-0.51264-0.19586-0.708 0l-4 4c-0.472 0.472 0.236 1.18 0.708 0.708l4-4c0.19586-0.19536 0.19586-0.51264 0-0.708z"
    />
    <path
      d="m5.3537 2.6459c-0.19536-0.19586-0.51264-0.19586-0.708 0l-2 2c-0.472 0.472 0.236 1.18 0.708 0.708l2-2c0.19586-0.19536 0.19586-0.51264 0-0.708z"
    />
    <path
      d="m12.363 7.6459c-0.19536-0.19586-0.51264-0.19586-0.708 0l-4 4c-0.472 0.472 0.236 1.18 0.708 0.708l4-4c0.19586-0.19536 0.19586-0.51264 0-0.708z"
    />
    <path
      d="m13.363 10.646c-0.19536-0.19586-0.51264-0.19586-0.708 0l-2 2c-0.472 0.472 0.236 1.18 0.708 0.708l2-2c0.19586-0.19536 0.19586-0.51264 0-0.708z"
    />
  </svg>
</div>
`;

exports[`Cursor 1`] = `
<div>
  <svg
    fill="currentColor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M14.082 2.182a.5.5 0 0 1 .103.557L8.528 15.467a.5.5 0 0 1-.917-.007L5.57 10.694.803 8.652a.5.5 0 0 1-.006-.916l12.728-5.657a.5.5 0 0 1 .556.103z"
    />
  </svg>
</div>
`;

exports[`Grid 1`] = `
<div>
  <svg
    fill="currentColor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M1 2a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2zm5 0a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H7a1 1 0 0 1-1-1V2zm5 0a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1h-2a1 1 0 0 1-1-1V2zM1 7a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V7zm5 0a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H7a1 1 0 0 1-1-1V7zm5 0a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1h-2a1 1 0 0 1-1-1V7zM1 12a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1v-2zm5 0a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H7a1 1 0 0 1-1-1v-2zm5 0a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1h-2a1 1 0 0 1-1-1v-2z"
    />
  </svg>
</div>
`;

exports[`LocationDot 1`] = `
<div>
  <svg
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      stroke-width="0.26458"
    >
      <ellipse
        cx="8"
        cy="8"
        fill="#6498fa"
        rx="8"
        ry="8"
      />
      <ellipse
        cx="8"
        cy="8"
        fill="#fff"
        rx="6"
        ry="6"
      />
    </g>
  </svg>
</div>
`;

exports[`Pause 1`] = `
<div>
  <svg
    fill="#FFF"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M5.5 3.5A1.5 1.5 0 0 1 7 5v6a1.5 1.5 0 0 1-3 0V5a1.5 1.5 0 0 1 1.5-1.5zm5 0A1.5 1.5 0 0 1 12 5v6a1.5 1.5 0 0 1-3 0V5a1.5 1.5 0 0 1 1.5-1.5z"
    />
  </svg>
</div>
`;

exports[`Play 1`] = `
<div>
  <svg
    fill="#FFF"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m11.596 8.697-6.363 3.692c-.54.313-1.233-.066-1.233-.697V4.308c0-.63.692-1.01 1.233-.696l6.363 3.692a.802.802 0 0 1 0 1.393z"
    />
  </svg>
</div>
`;
